docs/evaluate/development-production-features/multi-team.mdx

```diff
@@ -0,0 +1,94 @@
+---
+id: multi-team
+title: Multi-team - Temporal feature
```

Comment:
how did we come up with "Multi-team" as the header? Will our blog posts / other materials use Multi-team as the official term?

Reply:
We were recently talking about different names for the this that might capture Nexus better such as interservice communication? What do you suggest?

Reply:
would use
- Nexus - Temporal Feature (favor this one)
- Inter-service communication - Temporal feature

Reply:
I'll keep it as Temporal Nexus so the title matches the sidebar.